packages/okta-angular/test/e2e/harness/package.json

```diff
@@ -7,13 +7,11 @@
     "start": "yarn env && ng serve --port=3000",
     "env": "node ./scripts/prebuild.js",
     "build": "ng build",
-    "pretest": "./node_modules/.bin/webdriver-manager update --versions.chrome 2.46 --gecko false",
+    "pretest": "../../../../../scripts/updateSeDrivers.sh 0",
```

Comment:
Why do we need the "0" argument here?

Reply:
This was a sneaky one. i suppose i could have added a comment ... somewhere ... to explain it. It has to do with the 'workspace' configuration of the repo. some of the 'packages' depend on the `webdriver-manager` at the root level, and some depend on the one that's in the package. i'm passing in the extra param to tell it to use the local one.